### ISSUE TITLE
Feature: Get all contacts with active chats which are not existing contacts

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#571]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.79](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.78...1.9.79) (2020-06-24)

- build(deps): bump eventemitter2 from 6.4.2 to 6.4.3 [`#569`](https://github.com/open-wa/wa-automate-nodejs/pull/569)
- build(deps-dev): bump tsc-watch from 4.2.8 to 4.2.9 [`#570`](https://github.com/open-wa/wa-automate-nodejs/pull/570)
- build(deps-dev): bump @types/marked from 0.7.4 to 1.1.0 [`#567`](https://github.com/open-wa/wa-automate-nodejs/pull/567)
- Release 1.9.79 [`0649152`](https://github.com/open-wa/wa-automate-nodejs/commit/0649152f85e0dec0610bf6baabd294583b2e0bb6)
- Feature: Implemented safe mode #564 [`53236f0`](https://github.com/open-wa/wa-automate-nodejs/commit/53236f090ea9a83a7a3ca226f3aa72c695bd8f80)
- added prelim dockerfile, needs a lot of work #533 [`d80ca67`](https://github.com/open-wa/wa-automate-nodejs/commit/d80ca674b3b0c0914f18225191e87666ff7f859a)
- Feature: Implemented `getChatWithNonContacts` #571 [`f7e032b`](https://github.com/open-wa/wa-automate-nodejs/commit/f7e032bb47876214743b949e49ce56a39caa9e25)
